lab_1/main.py

```diff
@@ -4,19 +4,31 @@
 """
 
 
-def tokenize(text: str) -> list:
+def tokenize(text):
+    text = text.lower()
+    tokens = ''
+    signs = set ('.,!?@$:\/;%^&*[]{}+=-<>')
+    for i in text:
+        if i not in signs:
+            tokens += i
+    tokens = tokens.split()
+    return tokens
     """
     Splits sentences into tokens, converts the tokens into lowercase, removes punctuation
     :param text: the initial text
     :return: a list of lowercased tokens without punctuation
     e.g. text = 'The weather is sunny, the man is happy.'
     --> ['the', 'weather', 'is', 'sunny', 'the', 'man', 'is', 'happy']
     """
-    pass
+def remove_stop_words(tokens: list, stop_words: list) -> list:
+    for i in tokens:
+        if i in stop_words:
+            del tokens[tokens.index(i)]
```

Comment:
better not use del, can you suggest another way of doing that?

Reply:
try this:
tokens = ['the', 'a', 'is', 'the']
stop_words = ['the', 'a']

The expected result: ['is']